draft-wang-mmusic-encrypted-ice-candidates.md

```diff
@@ -242,14 +250,25 @@ Security Considerations {#security}
 =======================
 
 mDNS Message Flooding via Fallback Resolution
---------------------------------------------
+---------------------------------------------
 
 Encrypted candidates can be spoofed and signaled to an ICE agent to trigger the
 fallback mDNS resolution as described in step 6 in {{processing}}. This can
 potentially generate excessive traffic in the subnet. Note however that
 implementations of {{MdnsCandidate}} are required to have a proper rate
 limiting scheme of mDNS messages.
 
+Mutable ICE Password in Non-compliant JSEP Implementations
+--------------------------------------------------------
+
+Using the ICE password as the initialization parameter relies on its
+immutability between generation when creating the session description and
+consumption when gathering and encrypting candidates. {{JSEP}} prohibits the
+change of the session description returned from createOffer or createAnswer
+before being passed to setLocalDescription. A non-compliant JSEP implementation
+that allows such an interim change would enable a malicious application to
+choose the initialization parameter so as to compromise the underlying cipher.
```

Comment:
I think we should first detail the problem (including how the cipher is weakened) and then mention that JSEP protects against this.

Reply:
Done. PTAL.